Tests/secrets_white_list.json

```diff
@@ -22,6 +22,19 @@
   "iocs": {
     "ips": [
       "23.223.223.233",
+      "10.92.80.169",
```

Comment:
As mentioned in requested changes, check IPs added whether they're needed and move the needed IPs them to .sercets-ignore in HYAS pack